apiserver/facades/client/bundle/mock_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package bundle_test
+
+import (
+	"github.com/juju/testing"
+
+	"github.com/juju/description"
+)
+
+type mockState struct {
+	testing.Stub
+	model mockModel
```

Comment:
no need as an attribute here, just construct in export

Reply:
done